src/content-helper/common/providers/check-auth-provider.ts

```
@@ -0,0 +1,70 @@
+	 */
+	private static instance: CheckAuthProvider;
+
+	/**
+	 * Returns the singleton instance of the CheckAuthProvider.
+	 *
+	 * @since 3.19.0
+	 *
+	 * @return {CheckAuthProvider} The singleton instance.
+	 */
+	public static getInstance(): CheckAuthProvider {
+		if ( ! this.instance ) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)